src/lualib/NumberToFixed.ts

```diff
@@ -0,0 +1,13 @@
+/// https://www.ecma-international.org/ecma-262/10.0/index.html#sec-number.prototype.tofixed
+export function __TS__NumberToFixed(this: number, fractionDigits?: number): string {
+    if (Math.abs(this) >= 1e21 || this !== this) {
```

Comment:
I'm guessing this `this !== this` is to handle `NaN`? If so, please also add a test for `NaN`